index.js

```diff
@@ -115,8 +115,10 @@ class K8sVMExecutor extends Executor {
      * @param  {Number} [options.kubernetes.buildTimeout=90]          Number of minutes to allow a build to run before considering it is timed out
      * @param  {String} [options.kubernetes.resources.cpu.high=6]     Value for HIGH CPU (in cores)
      * @param  {Number} [options.kubernetes.resources.cpu.low=2]      Value for LOW CPU (in cores)
+     * @param  {Number} [options.kubernetes.resources.cpu.micro=0.5]  Value for LOW CPU (in cores)
```

Comment:
Should fix the comment Value for MICRO CPU (in cores)

Reply:
Thank you. I've fixed.